doc/interop-test-descriptions.md

```diff
@@ -418,21 +420,32 @@ Procedure:
       }
     }
     ```
+The client attaches custom metadata with the following keys:
+    ```
+    "x-grpc-test-echo-initial"
+    or
```

Comment:
The two keys refer to two different cases. In the first case the client is asking for initial metadata in the response stream , in the second case the client is asking for trailing metadata.